src/element/linearElementEditor.ts

```diff
@@ -72,15 +84,15 @@ export class LinearElementEditor {
       lastClickedPoint: -1,
     };
     this.hoverPointIndex = -1;
-    this.midPointHovered = false;
+    this.segmentMidPointHoveredCoords = null;
+    LinearElementEditor.updateEditorMidPointsCache(element, appState);
```

Comment:
unless I'm missing something, updating the cache anywhere is redundant because we already 
do it inside the getter, no?

Reply:
You are right, we don't need it anymore as now its updated on version change.